cabal-install/Distribution/Client/Setup.hs

```diff
@@ -1220,11 +1256,24 @@ installCommand = CommandUI {
      ++ " that, `configure` must be used.)\n"
      ++ " In contrast, without a sandbox, the flags to `install` are saved and"
      ++ " affect future commands such as `build` and `repl`. See the help for"
-     ++ " `configure` for a list of commands being affected.\n",
+     ++ " `configure` for a list of commands being affected.\n"
+     ++ "\n"
+     ++ "Installed executables will by default (and without a sandbox)"
+     ++ " be put into `~/.cabal/bin/`."
```

Comment:
I guess at this point in the code we don't have access to the config file. The default can be overridden in the config file (and indeed on OSX they often ship with a different default in the config file).

Perhaps we can do this. I think one of the other help options does specify the location of the config file (which is especially helpful to windows users), so perhaps we can do one better and read it to get the current settings and report them where needed in the --help.